server/services/prisonerProfileService.ts

```diff
@@ -113,17 +93,84 @@ export default class PrisonerProfileService {
       ]
     })
 
+    const supportedPrisons = await this.supportedPrisonsService.getSupportedPrisons(username)
+
+    const visitsForDisplay: visitItem[] = fullPrisoner.visits.map(visit => {
+      return [
+        {
+          html: `<a href='/visit/${visit.reference}'>${visit.reference}</a>`,
+          attributes: {
+            'data-test': 'tab-upcoming-reference',
```

Comment:
Is including `tab-upcoming-` in these various attributes here obsolete now that visits are all on one tab? E.g. perhaps prefixing with `tab-visits-` instead?